### ISSUE TITLE
[FEATURE] no-inline-styles option parameter to allow only dynamic styles

### ISSUE THREAD

**Issue by jpradelle** [yeonjuan/html-eslint#617]
Hi,
For rule `no-inline-styles`, I think it would be nice to have an option to allow only dynamic style in HTML done in template literals. By dynamic I mean any content containing a binding `${...}`

For example with the option enabled, this would be allowed
```javascript
let height = 150;
// ...
html`
  <foo style="height: ${height}px"></foo>
`
```

While this would still to be forbidden

```javascript
html`
  <foo style="height: 150px"></foo>
`
```

**Comment by yeonjuan** [comment]
@jpradelle  Thank you for the suggestion. It seems good to me
